Common/src/gRPC/ExpressionExt.cs

```diff
@@ -54,8 +55,8 @@ public static class ExpressionExt
   /// <param name="expr1"> The first predicate expression to combine </param>
   /// <param name="expr2"> The second predicate expression to combine </param>
   /// <returns> A new predicate expression that represents the logical OR of the two expressions </returns>
-  public static Expression<Func<T, bool>> ExpressionOr<T>(this Expression<Func<T, bool>> expr1,
-                                                          Expression<Func<T, bool>>      expr2)
+  public static Expression<Func<T, bool>>? ExpressionOr<T>(this Expression<Func<T, bool>> expr1,
```

Comment:
should  not return null
(you missed the factorization part)

Reply:
I was doing it . will push that part after ensuring the generic function works well

Reply:
Sorry, I misunderstood your comment . Done!